views/layout.pug

```diff
@@ -25,7 +25,10 @@ html(lang='en')
               li 
                 a(href='/catalog/bookinstances') All book-instances
               li 
+               a(href='/catalog/budgets') All budgets
```

Comment:
The indentation here is a bit weird -- just one space. (Everything else is two spaces.)